pkg/ttl/cache/table.go

```diff
@@ -189,24 +189,64 @@ func (t *PhysicalTable) ValidateKeyPrefix(key []types.Datum) error {
 func (t *PhysicalTable) EvalExpireTime(ctx context.Context, se session.Session,
 	now time.Time) (expire time.Time, err error) {
 	tz := se.GetSessionVars().Location()
+	now = now.In(tz)
```

Comment:
Would it be possible to only work with either time.Time or Unix epochs? as soon as one goes to strings or time representations with time zones it will add a lot of complexity and mental overhead.